core/src/epicli/data/common/ansible/playbooks/roles/node_exporter/tasks/install-node-exporter-as-system-service.yml

```diff
@@ -25,16 +25,17 @@
   vars:
     file_name: "{{ exporter_file_name }}"
 
-- name: Create /opt/node_exporter directories
+- name: Create node_exporter and textfile_collector directories
   file:
     path: "{{ item }}"
     recurse: true
     owner: root
     group: "node_exporter"
     mode: u=rwx,g=rx,o=
     state: directory
-  with_items:
+  loop:
```

Comment:
I would consider separate permissions, like:
```
  loop:
    - {path: /opt/node_exporter, mode: u=rwx,g=rx,o=}
    - {path: /var/lib/prometheus/node-exporter/textfile_collector, mode: u=rwx,g=rwx,o=rx}
```

since for `textfile_collector` directory I see [they use `755`](https://github.com/cloudalchemy/ansible-node-exporter/blob/c2401cfed554906843520cbac7ae2a3a132c590e/tasks/configure.yml#L40).

Reply:
This is done the same as in newer versions. If this is necessary to change, shoudn't it be fixed in latest and backported? But in general, I would not extend permissions until this is required.

Reply:
For standard metrics, I think you are right. For custom, we require `sudo` so being member of `node_exporter` group will not allow you to export custom metrics. Can be improved one day - maybe on request 😉